wic/pom.xml

```diff
@@ -9,32 +9,34 @@
     <packaging>war</packaging>
 
     <properties>
-        <org.slf4j-version>1.7.5</org.slf4j-version>
-        <org.springframework-version>5.1.2.RELEASE</org.springframework-version>
-        <spring-security-version>5.1.2.RELEASE</spring-security-version>
-        <spring.version>5.1.2.RELEASE</spring.version>
-        <spring-data.version>Lovelace-RELEASE</spring-data.version>
+        <org.slf4j-version>1.7.30</org.slf4j-version>
+        <org.springframework-version>5.2.9.RELEASE</org.springframework-version>
+        <spring-security-version>5.3.5.RELEASE</spring-security-version>
+        <spring.version>5.2.9.RELEASE</spring.version>
+        <spring-data.version>Neumann-RELEASE</spring-data.version>
+        <apache-tiles.version>3.0.8</apache-tiles.version> 
         <admin.password>admin</admin.password>
 
-        <hibernate.version>5.2.11.Final</hibernate.version>
+        <hibernate.version>5.4.22.Final</hibernate.version> <!-- 5.4.22.Final -->
         <hibernate.dialect>org.hibernate.dialect.MySQL5Dialect</hibernate.dialect>
         <hibernate.dialect.storage_engine>InnoDB</hibernate.dialect.storage_engine>
 
         <log.level>debug</log.level>
 
         <app.url>http://localhost:8080/wic/</app.url>
 
-        <quadriga.url>http://localhost:8080/quadriga/</quadriga.url>
+        <quadriga.url>https://diging.asu.edu/quadriga/</quadriga.url>
         <quadriga.concepts.endpoint>public/concepts</quadriga.concepts.endpoint>
-        <conceptpower.url>http://localhost:8080/conceptpower/rest/</conceptpower.url>
+        <!--<conceptpower.url>http://localhost:8080/conceptpower/rest/</conceptpower.url>-->
+        <conceptpower.url>https://chps.asu.edu/conceptpower/rest/</conceptpower.url>
```

Comment:
same here. your local settings should be in your settings.xml not changed in the pom